plugins/wu_palmer.py

```diff
@@ -119,6 +119,16 @@ class InputParametersSchema(FrontendFormBaseSchema):
             "input_type": "textarea",
         },
     )
+    root_has_meaning_in_taxonomy = ma.fields.Boolean(
+        required=False,
+        allow_none=False,
+        metadata={
+            "label": "Root node has meaning in the taxonomy",
+            "description": "Whether the value of the root node of a taxonomy has a meaning in the taxonomy"
+            "e.g. a taxonomy about colors and the root node represents a color",
```

Comment:
Vorschlag für eine andere Beschriftung: Consider root node part of the hierarchy.
Description wäre dann etwas wie: If the root node is part of the hierarchy, then items connected only through root are considered similar to a certain degree. Otherwise they will be considered as not similar.

Außerdem: String concatenation works in python without explicit + between literals but there must be whitespace in the strings... `...taxonomy" "e.g. a ...` should be `...taxonomy " ",e.g., a ...` (also missing commas around e.g.)